rendermodules/em_montage_qc/schemas.py

```diff
@@ -79,3 +80,44 @@ class DetectMontageDefectsParametersOutput(argschema.schemas.DefaultSchema):
         dtype='object',
         required=True,
         description='An array of (x,y) positions of seams for each section with seams')
+
+
+
+class RoughQCSchema(RenderParameters):
+    input_downsampled_stack = Str(
+        required=True,
+        description="Pre rough aligned downsampled stack")
+    output_downsampled_stack = Str(
+        required=True,
+        description="Rough aligned stack name")
+    minZ = Int(
+        required=True,
+        description="min z")
+    maxZ = Int(
+        required=True,
+        description="max z")
+    pool_size = Int(
+        required=False,
+        default=10,
+        missing=10,
+        description="Pool size")
+    output_dir = Str(
```

Comment:
Is there a reason not to make this an `OutputDir`?

Reply:
@mahaling This was unclear -- I actually meant  using `argschema.fields.OutputDir` as the field type -- I think "output_dir" as the name is fine.

Reply:
that is a classic confusion .... Will change it again...